setup.py

```diff
@@ -29,8 +31,8 @@ def __path(filename):
       url          = 'https://github.com/Omenia/robotframework-whitelibrary',
       install_requires = ['pythonnet', 'robotframework'],
       packages     = ['WhiteLibrary', 'WhiteLibrary.keywords', 'WhiteLibrary.keywords.items'],
-      package_dir  = {'WhiteLibrary' : 'src\WhiteLibrary'},
```

Comment:
These are considered as a escape sequeces if the string is not prefixed, but python itself does not distinquish \ or / as path serarator.